core/app/layout/layout.go

```diff
@@ -29,10 +30,15 @@ const (
 	ErrUnknownABI             = fault.Const("Unknown device ABI: %+v")
 )
 
+// LibraryType enumerates the possible GAPID dynamic libraries.
+type LibraryType int;
+const (
+	LibGraphicsSpy LibraryType = iota
```

Comment:
This PR needs gofmting. I would encourage you to try out VSCode with the go plugin - it's an awesome go editing environment and also automatically formats as you save.